<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAudiogramDialog</class>
 <widget class="QDialog" name="NewAudiogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewAudiogramDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QChartView" name="leftEarChart"/>
     </item>
     <item>
      <widget class="QChartView" name="rightEarChart"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Patient selection</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Patient ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="patientIDLE"/>
        </item>
        <item>
         <widget class="QPushButton" name="patientSelectPB">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Selected sample</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="firstSampleRB">
          <property name="text">
           <string>First</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="secondSampleRB">
          <property name="text">
           <string>Second</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="leftEarPB">
         <property name="text">
          <string>Edit LEFT ear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rightEarPB">
         <property name="text">
          <string>Edit RIGHT ear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>NewAudiogramDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>NewAudiogramDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
